XML/Other/ReplaceableTestParameters.xml

```diff
@@ -71,11 +71,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>VPP_SOURCE_BRANCH</ReplaceThis>
-		<ReplaceWith>stable/1901</ReplaceWith>
-	</Parameter>
-	<Parameter>
-		<ReplaceThis>VPP_DPDK_VERSION</ReplaceThis>
```

Comment:
Those are two different parameters? What makes you remove the second one?

Reply:
During testing, when build VPP, no need to specify the dpdk version.